tests/scattering1d/test_torch_backend_1d.py

```diff
@@ -12,13 +12,14 @@
         skcuda_available = True
 except:
     Warning('torch_skcuda backend not available.')
+skcuda_available = False
```

Comment:
Why? Where's the `torch_skcuda` backend anyhow?

Reply:
I'll be pushing it soon enough.